runtime/doc/lua.txt

```diff
@@ -3897,6 +3900,7 @@ Iter:enumerate()                                            *Iter:enumerate()*
 <
 
     Example: >lua
+
```

Comment:
Why add this empty new line at the beginning of code blocks? Is it intended?

Reply:
I made a minor change in text_utils.lua to not strip newlines in code blocks.

The stripping was added in the initial commit for text_utils.lua to minimise the diff.